_topics/en/schnorr-signatures.md

```diff
@@ -98,6 +98,12 @@ optech_mentions:
   - title: Proposal to update BIP340 schnorr signatures to use evenness tiebreaker
     url: /en/newsletters/2020/08/19/#proposed-uniform-tiebreaker-in-schnorr-signatures
 
+  - title: Updates BIP340 to consistently use evenness tiebreaker
```

Comment:
Some of the links for BIPs PRs include the PR number and some don't. Should we aim for a consistent style?

Reply:
Agree that I should be consistent about including PR numbers, but I think we should allow exceptions (in particular, sometimes it can be a challenge to get the number and description to both fit within our max character allowance for titles).